Assets/Scripts/Actors/BuddyStats.cs

```diff
@@ -30,19 +41,9 @@ void Awake()
 	{
 		ID = GetID(); // Grab the current unique ID
 		name = "Buddy " + GetRandomName( ID );
-		currResourceTimer = decreaseResourcesTime;
 		apples = startingApples;
 		_particles = GetComponentInChildren<ParticleSystem>();
-		SetGod(owner); // For owners set in the inspector
-	}
-
-	public void SetGod( GodTag _godTag )
-	{
-		if(_godTag)
-		{
-			owner = _godTag;
-			_actorStats = _godTag.gameObject.GetComponent<PlayerActorStats>();
-		}
+		owner = _startingOwner; // For owners set in the inspector
```

Comment:
Can we just get the god in the scene for this so we don't have to manually assign it for every buddy in the scene? We can change it later if we get to other gods.